apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-a-multiple-entries-same-bene.txt

```diff
@@ -0,0 +1,6 @@
+INSERT|567834|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|543217066U|Doe|John|A|3456789|17-Mar-1981|1990|4|5|6|7|8|A|B|C|D|4|19631003|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|1|1|1|1|1|1|1|1|1|1|0|0|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|0|0|003|003|003|003|003|003|003|003|003|003|||00|00|00|00|00|00|00|00|00|00|||S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|0|003|003|003|003|003|003|003|003|003|003|003||000|000|000|000|000|000|000|000|000|000|000||Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|N|**|**|**|**|**|**|**|**|**|**|**|**|00|00|01|01|05|06|07|08|01|02|00|00|204 SOUTH ST|7560 123TH ST|SURREY|DAEJEON SI 34867|COLOMBIA|SURREY|PODUNK|IA|123456789|30-JUL-2020||1
```

Comment:
I think the use case is all UPDATEs as we don't expect INSERTs and UPDATEs spread across multiple batches to necessarily be handled.

Reply:
I added that first insert so the bene existed to update, but given your comment I adjusted the tests to have an initial load (with no ref year so it didnt populate bene history) and then 6 updates in the file.